recipes/pytest-randomly/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pytest-randomly" %}
+{% set version = "3.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-randomly-{{ version }}.tar.gz
+  sha256: 440cec143fd9b0adeb072006c71e0294402a2bc2ccd08079c2341087ba4cf2d1
+
+build:
+  number: 0
+  skip: true   # [py<36]
```

The skip can be removed and this can be noarch:
```suggestion
  noarch: python
```

---

@dbast, this package has python version-dependent requirements, which ai think precludes it from being a noarch:python package.

---

That can be handled we min python version run requirement. see other comments/suggestions

---

@dbast, sorry for the delay on this - my understanding is that this

```yaml
    - importlib-metadata  # [py<38]
```

means that this package cannot be `noarch:python`. Has that been changed?

---

I see at https://conda-forge.org/docs/maintainer/knowledge_base.html?#noarch-python that

>     - No python version specific requirements

is still listed.

---

Yes, that's still the case. But if that's the only reason not to make this noarch, it's better to just remove the version selector on `importlib-metadata`.